### DISCUSSION TITLE
Where are changes saved

### DISCUSSION THREAD

**Discussion by Timonade** [streetcomplete/StreetComplete#5250] [Q&A] [answered]
Where are the changes saved before I upload them? I can't find them anywhere.

Why am I looking for the changes?
- I have a new phone and would upload the changes collected from one place. My problem new phone all changes gone.

**Comment by mnalis** [comment, marked as answer]
In internal SQLite database. Unless your phone is rooted, you won't be able to see / access that folder.

You should upload your changes on the old phone.
